acceptance/openstack/waf/v1/domains_test.go

```diff
@@ -36,8 +37,8 @@ func preparePolicy(t *testing.T, client *golangsdk.ServiceClient) *policies.Poli
 func prepareCertificate(t *testing.T, client *golangsdk.ServiceClient) *certificates.Certificate {
 	cert, err := certificates.Create(client, certificates.CreateOpts{
 		Name:    "waf_cert_1",
-		Content: "-----BEGIN CERTIFICATE-----MIIDIjCCAougAwIBAgIJALV96mEtVF4EMA0GCSqGSIb3DQEBBQUAMGoxCzAJBgNVBAYTAnh4MQswCQYDVQQIEwJ4eDELMAkGA1UEBxMCeHgxCzAJBgNVBAoTAnh4MQswCQYDVQQLEwJ-----END CERTIFICATE-----",
-		Key:     "-----BEGIN RSA PRIVATE KEY-----MIICXQIBAAKBgQDFPN9ojPndxSC4E1pqWQVKGHCFlXAAGBOxbGfSzXqzsoyacotueqMqXQbxrPSQFATeVmhZPNVEMdvcAMjYsV/mymtAwVqVA6q/OFdX/b3UHO+b/VqLo3J5SrM-----END RSA PRIVATE KEY-----",
+		Content: testCert,
+		Key:     testKey,
 	}).Extract()
 	if err != nil {
```

Comment:
Can you change all `if err` to `th.Assert`?

Reply:
Didn't want to touch too much code. Want to change only certificate-related code.